src/app/GitCommands/RevisionReader.cs

```diff
@@ -334,6 +340,34 @@ public void GetLog(
             }
         }
 
+        private static void AddAutoStash(string workingDirGitDir, IObserver<IReadOnlyList<GitRevision>> subject, string autostashLabel)
+        {
+            string autoStashFileName = Path.Combine(workingDirGitDir, "rebase-merge/autostash");
+            if (!File.Exists(autoStashFileName)
+                || !ObjectId.TryParse(File.ReadLines(autoStashFileName).FirstOrDefault(), out ObjectId? autoStashCommitId))
+            {
+                return;
+            }
+
+            long now = DateTimeUtils.ToUnixTime(File.GetLastWriteTime(autoStashFileName));
+            GitRevision autoStashRevision = new(autoStashCommitId)
+            {
+                AuthorUnixTime = now,
```

Comment:
Add `Author = "git",`?

Reply:
It is still the current user that makes the commit (autostash)
I rather have that blank:https://github.com/gitextensions/gitextensions/pull/12200#discussion_r1958789319